custom_decorator/activity_utils.py

```diff
@@ -17,7 +17,7 @@ async def wrapper(*args, **kwargs):
         # Heartbeat twice as often as the timeout
         heartbeat_timeout = activity.info().heartbeat_timeout
         if heartbeat_timeout:
-            asyncio.create_task(
+            _ = asyncio.create_task(
```

Comment:
The right answer is technically probably to capture this and cancel-then-await it in a finally instead of using the done event, but this probably works too.

May deserve a comment though on why you added this. Also I assume we can trust the Python interpreter will never get smart enough to know the `_` var can be collected since it is not referenced anymore?

Reply:
Great call, the cancel-and-await strategy is not only more stable, but doesn't leave a confusing `_` that would require some explanation.

Do you recommend I similarly adjust my approach for `compose_greeting()`, or just add a comment for that one?

Reply:
Can just add comment if you want there (the lifetime of that `_` is probably not long enough for that use case anyways). I think your latest update is plenty elegant.

Reply:
Thanks, added.  Please forgive the plagiarism! 😅